### PR TITLE
Extract geo extension center point data

### PR DESCRIPTION
**Pull request by cbeer**
Part of https://github.com/sul-dlss/exhibits/issues/416

### CODE REVIEW

File: lib/stanford-mods/geo_spatial.rb [hunk 1]
```diff
@@ -32,6 +32,22 @@ def geo_extensions_as_envelope
...
+        mods_ng_xml.extension
+                   .xpath(
+                     '//rdf:RDF/rdf:Description/gmd:centerPoint/gml:Point[gml:pos]',
+                     'gml' => GMLNS,
+                     'gmd' => 'http://www.isotc211.org/2005/gmd',
+                     'rdf' => 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+                   ).map do |v|
+                     lat, long = v.xpath('gml:pos', 'gml' => GMLNS).text.split
+                     "#{long} #{lat}"
+                   end
```
Comment (atz) [inline lib/stanford-mods/geo_spatial.rb]: Are you uninterested in providing the same exception protection as the preceding method?

Comment (cbeer) [inline lib/stanford-mods/geo_spatial.rb]: What exceptions need to be handled? I couldn't think of any.

Comment (cbeer) [inline lib/stanford-mods/geo_spatial.rb]: And this morning, I wrote some tests for a variety of cases including:

- no `<extension>` elements
- no `<gmd:centerPoint>` element
- no `<gml:Point>` element
- no `<gml:pos>` element
- malformed data in the `<gml:pos>`, including schema-invalid data with too many elements, too few elements, or no text content at all.

Looking in #87, where the exception handling code for `geo_extensions_as_envelope` was added, the only test simulate an exception, but, again, it's unclear to me what real-world scenario we're guarding against.

Comment (jkeck) [inline lib/stanford-mods/geo_spatial.rb]: Unless we can provide a test case where a `RuntimeError` will be raised (that's not explicitly raising one in the test itself), it seems like this should be okay to 🚢.

Comment (atz) [inline lib/stanford-mods/geo_spatial.rb]: OK, cool, as long as you have a reasonable degree of confidence that it is robust.  Thanks for checking.

**Pull request merged**